geps/gep-713/index.md

```diff
@@ -109,395 +114,67 @@ additional fields that are _not_ in the `spec` of the object.
 
 ### Direct Policy Attachment
 
-A Direct Policy Attachment is tightly bound to one instance of a particular
-Kind within a single namespace (or to an instance of a single Kind at cluster scope),
-and only modifies the behavior of the object that matches its binding.
+For more description of the details of Direct Policy Attachment,
+see [GEP-2648](https://gateway-api.sigs.k8s.io/geps/gep-2648/).
 
-As an example, one use case that Gateway API currently does not support is how
-to configure details of the TLS required to connect to a backend (in other words,
-if the process running inside the backend workload expects TLS, not that some
-automated infrastructure layer is provisioning TLS as in the Mesh case).
+### Inherited Policy Attachment
```

Comment:
I imagine this has already been discussed ad nauseam, but why not name `Inherited` Policy Attachment as `Multiple` or `Indirect`?

Reply:
"Inherited" is a closer match to what's actually happening. I do find myself using "Indirect" sometimes though, so it may well end up as an accepted synonum.